corehq/apps/users/__init__.py

```diff
@@ -1,4 +1,17 @@
 from corehq.preindex import ExtraPreindexPlugin
+from django.apps import AppConfig
 from django.conf import settings
 
-ExtraPreindexPlugin.register('users', __file__, settings.NEW_USERS_GROUPS_DB)
+from .signals import connect_user_signals
+
+
+class UsersAppConfig(AppConfig):
+    name = 'corehq.apps.users'
+
+    def ready(self):
+        """Code to run with Django starts"""
+        ExtraPreindexPlugin.register('users', __file__, settings.NEW_USERS_GROUPS_DB)
```

Comment:
@dannyroberts FYI - I took the liberty of moving this into `AppConfig.ready()`.  Know of any reason why that's not a good idea?